test/models/concerns/audit_extensions_test.rb

```diff
@@ -41,4 +41,9 @@ def setup
     FactoryGirl.create :lookup_value, :lookup_key_id => key.id, :value => false, :match => "hostgroup=Common"
     refute_empty Audit.search_for("type = lookupvalue")
   end
+
+  test "search for type=compute_resource in audit" do
+    FactoryGirl.create(:ec2_cr)
+    refute_empty Audit.search_for("type = compute_resource")
```

Comment:
`assert_empty` before the creation please to make sure that it isn't empty because the creation added an entry.